js/server/assets/swagger/index.html

```diff
@@ -86,8 +86,8 @@
         ],
         layout: "StandaloneLayout",
         //#region ArangoDB-specific changes
-        // Expand the API documentation by default.
-        docExpansion: "list",
+        validatorUrl: null,
+        docExpansion: "none",
```

Comment:
hm, I vaguely remember that we didn't want to change this before 3.7 - but I personally don't care.

Reply:
Frank and Heiko are okay with the change and I think it's for the better.

Reply:
Can we please keep this as-is as it also affects Foxx? See the other PRs for discussion.

Reply:
Restored to "list".